stdlib/LLVMLibUnwind_jll/src/LLVMLibUnwind_jll.jl

```diff
@@ -14,18 +14,18 @@ export llvmlibunwind
 # These get calculated in __init__()
 const PATH = Ref("")
 const LIBPATH = Ref("")
-artifact_dir = ""
-llvmlibunwind_handle = C_NULL
-llvmlibunwind_path = ""
+artifact_dir::String = ""
+llvmlibunwind_handle::Ptr{Cvoid} = C_NULL
+llvmlibunwind_path::String = ""
 
 const llvmlibunwind = "libunwind"
 
 function __init__()
     # We only dlopen something on MacOS
     @static if Sys.isapple()
-        global llvmlibunwind_handle = dlopen(llvmlibunwind)
-        global llvmlibunwind_path = dlpath(llvmlibunwind_handle)
-        global artifact_dir = dirname(Sys.BINDIR)
+        llvmlibunwind_handle = dlopen(llvmlibunwind)
+        llvmlibunwind_path = dlpath(llvmlibunwind_handle)
+        artifact_dir = dirname(Sys.BINDIR)
```

Comment:
These need to have the `global`, otherwise they will only be local variables here.

Reply:
I've pushed another commit, you can run it now without using global variables.

Also, changes:

- We have added a `isapple()` function that looks like this:
```jl
isapple() = Sys.isapple()
```
- We have added a check in `is_available()` to see if it is an apple device. If it is an apple device, then it is available.
- Likewise, if it is an apple device, then we have an artifact directory. If it is not an apple device, then we dont have an artifact directory.
- If it is an apple device, then there is a path to the library. If it is not an apple device, then there is not a path to the library.

Reply:
No, these are still just local variables in the init function.

Reply:
You can just revert this whole section of the change.